ui/raidboss/timeline.ts

```diff
@@ -443,7 +542,7 @@ export class Timeline {
       this.syncTimeCallback(fightNow, false);
   }
 
-  SyncTo(fightNow, currentTime) {
+  protected SyncTo(fightNow: number, currentTime: number): void {
```

Comment:
I don't feel super strongly about protected vs public.  Private at least has some immediate meaning that says "you can see all of the uses in this file (probably)".  Protected is like "you probably shouldn't use this I guess, but who knows how it's actually being used".